webserver/website/src/views/stats/Stats.vue

```diff
@@ -1,212 +1,21 @@
-          <div class="departure">
-            <span>{{ stats.all_perc_dp_cancel.toLocaleString('de-DE') }}%</span>
-          </div>
-
-          <div class="rowspan divider"></div>
-
-          <!-- Most recent day of data -->
-          <div class="vertical-header">
-            <div class="vertical-rl">Daten vom {{ stats.new_date }}</div>
-          </div>
-          <div class="stat header">Anzahl an Halten</div>
-          <div class="arrival">
-            <span>{{ stats.new_num_ar_data.toLocaleString('de-DE') }}</span>
-          </div>
-          <div class="departure">
-            <span>{{ stats.new_num_dp_data.toLocaleString('de-DE') }}</span>
-          </div>
-
-          <div class="stat header">Maximale Verspätung</div>
-          <div class="arrival">
-            <span>{{ stats.new_max_ar_delay.toLocaleString('de-DE') }} min</span>
-          </div>
-          <div class="departure">
-            <span>{{ stats.new_max_dp_delay.toLocaleString('de-DE') }} min</span>
-          </div>
-
-          <div class="stat header">Ø Verspätung</div>
-          <div class="arrival">
-            <span>{{ stats.new_avg_ar_delay.toLocaleString('de-DE') }} min</span>
-          </div>
-          <div class="departure">
-            <span>{{ stats.new_avg_dp_delay.toLocaleString('de-DE') }} min</span>
-          </div>
-
-          <div class="stat header">Prozent an Verspätungen</div>
-          <div class="arrival">
-            <span>{{ stats.new_perc_ar_delay.toLocaleString('de-DE') }}%</span>
-          </div>
-          <div class="departure">
-            <span>{{ stats.new_perc_dp_delay.toLocaleString('de-DE') }}%</span>
-          </div>
-
-          <div class="stat header">Prozent an Ausfällen</div>
-          <div class="arrival">
-            <span>{{ stats.new_perc_ar_cancel.toLocaleString('de-DE') }}%</span>
-          </div>
-          <div class="departure">
-            <span>{{ stats.new_perc_dp_cancel.toLocaleString('de-DE') }}%</span>
-          </div>
-        </div>
-      </div>
+  <div id="about" class="text_content">
+    <h1 class="text-center">Statistiken</h1>
+    <div>
+      Wer wollte alles schon mal wissen wie pünktlich die Bahn gestern war? Oder welche Zugstrecken besonders
+      zuverlässig sind? Wir sammeln rund um die Uhr die Verspätungsinformationen aus den Bahnverkehrsdaten, um diese in
+      unsere Verbindungsvorhersage zu integrieren. Hier an dieser Stelle veröffentlichen wir auch unsere selbst
+      erstellten Statistiken zur Pünktlichkeit im Bahnverkehr.
+    </div>
+    <div class="d-flex justify-content-center gap-5">
+      <router-link class="btn btn-primary" :to="{ path: '/stats/overview', hash: '#content' }">Übersicht</router-link>
+      <router-link class="btn btn-primary" :to="{ path: '/stats/stations', hash: '#content' }">Bahnhöfe</router-link>
+    </div>
+    <div>
+      Vorsicht: Alle diese Daten sind eine Beobachtung von dritten und nicht von der Bahn selber. Wir können die
```

Comment:
Ich würde auch des hier noch umbennen zu `Disclaimer` und in einem anderen Font machen oder so, sodass des vom Info text differenziert wird